card/lib/card/env.rb

```diff
@@ -45,19 +45,12 @@ def slot_opts
         # FIXME:  upgrade to safe parameters
         self[:slot_opts] ||= begin
           opts = params[:slot] || {}
-          opts.merge shortcut_slot_opts
+          opts.merge! shortcut_slot_opts
```

Comment:
important bang.

Reply:
with this fixed, we can get rid of all the other references to `params[:item]`

Reply:
If you want to manipulate params maybe you should do it less concealed.

Reply:
I don't think we need to manipulate params.  I'll change this to clone.